### PR TITLE
TASK: updated travis.yml | removed any "typo3" | added all dependencies

### PR DESCRIPTION
**Pull request by ComiR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,15 +28,11 @@ composer require 'flowpack/elasticsearch-contentrepositoryadaptor'
...
-Now, add the routes as described in the [README of Flowpack.SearchPlugin](https://github.com/Flowpack/Flowpack.SearchPlugin)
-as the **first route** in Configuration/Routes.yaml.
+Then just run `./flow nodeindex:build` and add the search plugin to your page. It should "just work".
 
-Then, ensure to update `<your-elasticsearch>/config/elasticsearch.yml` as explained below; then start Elasticsearch.
-
-Finally, run `./flow nodeindex:build`, and add the search plugin to your page. It should "just work".
-
-## Elasticsearch Configuration file elasticsearch.yml
+## Elasticsearch Configuration file *elasticsearch.yml*
```
Comment (kdambekalns) [inline README.md]: So, what's with the capitalisation? ;)

File: composer.json [hunk 2]
```diff
@@ -3,22 +3,26 @@
...
-        "neos/content-repository-search": "~3.0 || dev-master",
-        "neos/flow": "~4.0 || dev-master",
-        "neos/fluid-adaptor": "~1.0 || dev-master"
+        "flowpack/elasticsearch": "^2.0",
+        "neos/content-repository": "^3.0",
+        "neos/content-repository-search": "^3.0",
+        "neos/eel": "^4.0",
+        "neos/flow": "^4.0",
+        "neos/fluid-adaptor": "^4.0",
+        "neos/utility-arrays": "^4.0"
```
Comment (kdambekalns) [inline composer.json]: Some of those are required by Flow anyway, I'd not require them explicitly (again). After all, in Flow we  require only `doctrine/orm`, not all it's transitive dependencies…

But, I know there are differing views regarding that, so: was this done on purpose?

File: composer.json [hunk 3]
```diff
@@ -3,22 +3,26 @@
...
-    ],
+    "license": "LGPL-3.0",
     "require": {
         "php": ">=7.0",
-        "flowpack/elasticsearch": "~2.0 || dev-master",
-        "neos/content-repository-search": "~3.0 || dev-master",
-        "neos/flow": "~4.0 || dev-master",
-        "neos/fluid-adaptor": "~1.0 || dev-master"
+        "flowpack/elasticsearch": "^2.0",
+        "neos/content-repository": "^3.0",
```
Comment (kdambekalns) [inline composer.json]: Isn't this required by `neos/content-repository-search` already? See next comment, too…

File: composer.json (hunk 2 above)
Comment (ComiR) [inline composer.json]: Yes - see https://discuss.neos.io/t/composer-dependencies-declare-transitive-dependencies/1953.

File: README.md (hunk 1 above)
Comment (ComiR) [inline README.md]: yeah, should've corrected that too ;)

**Pull request closed without merging**